src/lib/internal/helpers/tests/array.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, expect, test } from 'vitest';
```

Comment:
Oh it's duplicated 😄

Reply:
Ahh yep great catch! I deleted the dupe :)